### PR TITLE
remove dependencies and add css for datatable.

### PR DESCRIPTION
**Pull request by yash013**
removed these dependencies 
- angular-datatables
- datatables.net
- datatables.net-dt

added CSS for the DataTable in HelpComponent.

### CODE REVIEW

File: ui/src/app/editor/dialog/help/help.component.html [hunk 1]
```diff
@@ -16,6 +16,29 @@
...
+    }
+
+    .table tbody tr:hover {
+        font-weight: bold;
+        background-color: #f1ecec;
+    }
+
+    
+</style>
+
```
Comment (dominikriemer) [inline ui/src/app/editor/dialog/help/help.component.html:41]: Styles should be added to the component's scss file (see help.component.scss) and not directly in the HTML. Style tags are typically used in the HTML header section only.

See https://angular.io/guide/component-styles#style-files-in-component-metadata for some info on how styles can be included in Angular components.

**Review by dominikriemer — CHANGES_REQUESTED** [review]

**Comment by tenthe** [comment]
Thank you very much for the PR.
Can you please move the css code directly into the .scss file as @dominikriemer mentioned.

**Comment by tenthe** [comment]
Thanks for the update! I had a look at the PR and have some minor comments:
- Did StreamPipes start for you? 
	- I had to remove `"node_modules/datatables.net/js/jquery.dataTables.js"` in the file angular.json
- The **fields tab** looks good but the table in **live preview** still has the old `dataTable` CSS class.
	- You can find the implementation in the component `PipelineElementRuntimeInfoComponent`
	- Can you please update this component as well?

Please also merge the latest version of the `dev` branch into your PR, sinc we fixed the tests there yesterday.

**Comment by yash013** [comment]
sure. i will do that

**Comment by yash013** [comment]
i was adding some css to the preview.component but it is not working .That changes worked in HelpComponent.

**Comment by tenthe** [comment]
Did you add it to this file? `./pipeline-element-runtime-info.component.scss`
The table for the priview should be located in `connect/components/runtime-info/pipeline-element-runtime-info.component.html`

**Comment by yash013** [comment]
review please

**Comment by tenthe** [comment]
Thanks a lot, I'll have a look at it

**Comment by bossenti** [comment]
@yash013 thanks a lot for your work so far 🙏🏼 
It looks like you haven't set up the pre-commit hook for our UI.
You can read here how to do this: https://github.com/apache/streampipes/blob/dev/ui/README.md

**Comment by bossenti** [comment]
@yash013 are you still working on this PR?
Otherwise, we will take over it or even close it
In case you want to pick it up somewhen, you can always just reopen it

**Comment by yash013** [comment]
Actually I'm very busy in some other business project 
 I will surely complete that whenever I get time. For now I'm not working on this pr hope you understand.

**Comment by bossenti** [comment]
Sure, no problems :) 
Just wanted to keep up with our PRs

**Pull request closed without merging**